### ISSUE TITLE
Incorrect alert in Layout/IndentationConsistency

### ISSUE THREAD

**Issue by tas50** [rubocop-hq/rubocop#8049]
## Expected behavior

Layout/IndentationConsistency should ensure that all values with the body of a case statement are aligned with each other.

## Actual behavior

Layout/IndentationConsistency incorrectly autocorrect one of the when bodies to have an addition 2 spaces of indentation.

## Steps to reproduce the problem

Run rubocop against this code:

```ruby
opts[:ssl_verify_mode] = case new_resource.ssl_verify_mode
                         when :verify_none
                           Chef::HTTP::VerifyNoneSSLPolicy
                         else :verify_peer
                           Chef::HTTP::VerifyPeerSSLPolicy
                         end
```

This will produce this warning

```
test.rb:7:28: C: Layout/IndentationConsistency: Inconsistent indentation detected.
                           Chef::HTTP::VerifyPeerSSLPolicy
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

This will autocorrect to:

```ruby
# frozen_string_literal: true

opts[:ssl_verify_mode] = case new_resource.ssl_verify_mode
                         when :verify_none
                           Chef::HTTP::VerifyNoneSSLPolicy
                         else :verify_peer
                              Chef::HTTP::VerifyPeerSSLPolicy
                         end
```

## RuboCop version

```
master (0.84)
```

**Comment by tas50** [comment]
🤦 Ignore me entirely today
